DISC-NU/books-and-breakfast-app
# Add userinfo real time listener and set userinfo context to database …

### Description

Adds firebase function to subscribe to changes in user info. Adds logic to set user info context based on changes in the database when user exists and is registered. Now all users are logged out when their data is cleared (when admin clicks wipe button)

Makes Navigation Screen and Tips Screen take the values of schoolName and isAdmin from userInfo context instead of route from Home Screen so that both screens re-renders immediately on value change

### QA

Please explain in detailed steps how the reviewer can test your changes and what the expected behaivors are (visual would help!). Consider different test cases/scenarios.

As a registered user, make yourself admin and click DELETE ALL USERS button, check that you are logged out and have to reselect all values again. OR just delete your user data from firebase and check that you are logged out

### Packages

N/A

### Checklist
